cheroot/server.py

```diff
@@ -1866,8 +1868,27 @@ def prepare(self):  # noqa: C901  # FIXME
         self.ready = True
         self._start_time = time.time()
 
+    def _serve_unservicable(self):
+        """Serve connections we can't handle a 503."""
+        while self.ready:
+            conn = self._unservicable_conns.get()
+            if conn is None:
+                return
+            request = HTTPRequest(self, conn)
+            try:
+                request.simple_response('503 Service Unavailable')
+            except Exception as ex:
+                self.server.error_log(
+                    repr(ex),
+                    level=logging.ERROR,
+                    traceback=True,
+                )
+            conn.linger = True
```

Comment:
Why is this necessary?

Reply:
I was just copying existing code that did that in error handling, but empirically looks like not on Linux at least. Will see what CI does on Windows and macOS.